core/src/main/scala/org/broadinstitute/dsde/rawls/util/WorkspaceSupport.scala

```diff
@@ -186,6 +186,7 @@ trait WorkspaceSupport {
                                         attributeSpecs: Option[WorkspaceAttributeSpecs] = None
   ): Future[Workspace] =
     for {
+      _ <- userEnabledCheck(ctx.userInfo)
```

Comment:
thinking out loud: I poked at this a bit to find a good way to avoid always making this Sam call. The best I could come up with was adding a recover/catch and only making this extra Sam call on an error, which would eliminate the extra Sam call from the typical/successful path. The code gets more complex with that. Again, not gonna block the PR on this, not requesting changes, just talking.